tests/functional/d/deprecated/deprecated_module_py3.rc

```diff
@@ -1,2 +1,5 @@
 [typing]
 py-version=3.6
+
+[testoptions]
+exclude_from_minimal_messages_config=true
```

Comment:
Just wondering, should we do this the other way around: allowing certain tests to be "optimised" by this option? Or would that be too much work?

Reply:
No, I would leave it this way:
This check makes (imo) a lot of sense, and it should be enabled by default (once we found a suitable place for it in the CI). 
Being able to check for just a single message explicitly is not an optimisation for me, but part of "correct functional behaviour". Doing this is even mentioned in [our docs](https://pylint.pycqa.org/en/latest/user_guide/run.html?highlight=disable%3Dall#command-line-options).

Reply:
Just to be sure, we're also testing that the message is emitted when everything is activated like we did before ?

Reply:
Yes. This option is not enabled by default. Right now, the CI doesn't use it in any way yet. 
We should add a separate test job for it, because testing for false positives could be tricky if we made this the default behavior.

Reply:
That's why I wonder if we should make this opt-in instead of opt-out. That way we can just enable this in the general CI job and opt tests in whenever we want to?

Reply:
I think the default use of pylint is to check all messages so opt out is the best way to add --minimal-message-config  imo.

Reply:
Daniel, maybe we are talking about different things? 
Right now, all test runs will behave exactly the same as before. Only if you run ``pytest --minimal-messages-config tests/test_functional.py``, all messages except the ones which are checked by the test are disabled.

Ideally, one day we can run all of the functional tests once with all messages enabled and once with only the specific message enabled. This way we can check both: undesired side effects from code executed for other messages, and undesired side effects due to code *not* being run because messages are disabled.
Both could lead to bugs.

Reply:
Hm okay. Yeah I was thinking of one day running just once with this flag enabled, but the way you put it it might actually be better to do both. Then this system is fine!